src/index.js

```diff
@@ -23,6 +23,7 @@ export default class Undo {
 
     const { configuration } = editor;
 
+    this.holder = typeof configuration.holder === 'string' ? document.getElementById(configuration.holder) : configuration.holder;
```

Comment:
@MarioRodriguezS you can use object destructing here

```javascript
const { holder } = configuration;

this.holder = typeof holder === 'string' ? document.getElementById(holder) : holder;
```

Reply:
Done!. Thanks